observability/micrometer/src/main/resources/schema/wildfly-micrometer_1_0.xsd

```diff
@@ -26,17 +26,24 @@
     <!-- The subsystem root element -->
     <xs:element name="subsystem">
         <xs:complexType>
-            <xs:attribute name="security-enabled" type="xs:boolean" default="true">
+            <xs:attribute name="exposed-subsystems" type="xs:string">
                 <xs:annotation>
                     <xs:documentation>
-                        True if authentication is required to access the HTTP endpoint on the HTTP management interface.
+                        The names of the WildFly subsystems that exposes their metrics (or '*' to expose any subsystem metrics).
                     </xs:documentation>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="exposed-subsystems" type="xs:string">
+            <xs:attribute name="endpoint" type="xs:string" default="http://localhost:4318/v1/metrics">
```

Comment:
The defaultvalue is now inconsistent with `org.wildfly.extension.micrometer.MicrometerSubsystemDefinition#ENDPOINT`. :-(